app/src/models/subscription.js

```diff
@@ -46,9 +46,7 @@ const Subscription = new Schema({
     },
     createdAt: { type: Date, required: true, default: Date.now },
     updateAt: { type: Date, required: false, default: Date.now },
-    application: {
-        type: String, required: true, default: 'gfw', trim: true
-    },
+    application: { type: String, trim: true },
```

Comment:
we have to revert this (as much as it pains me)

Reply:
Also: we should advertise https://github.com/resource-watch/doc-api/compare/367a648aac7d...86add3cb1230 on WRI channels, so people start making the necessary adjustments, so that we can one day revisit this change.

Reply:
Reverted the change for subscriptions (statistics are only created via code, so we have the control over that).